script/release/01.push-release-branches-DSL.txt

```diff
@@ -0,0 +1,66 @@
+def cloneScript_01 = 
```

Comment:
@psiroky 
I think it could be a benefit if we have the DSL script stored somewhere in order not having to redefine it when this Job get lost.
I think it is a good idea to change all hard-coded path to a placeholder,